test/box/transaction.result

```diff
@@ -234,6 +234,10 @@ while f:status() == 'running' do
 end;
 ---
 ...
+box.commit();
```

Comment:
Removed it. The problem was that the commit wasn't called here:

https://github.com/tarantool/tarantool/blob/1c6b6f85f4d36787f9e0a861a5437412e65bae11/test/box/transaction.result#L214-L217

Note that there's no delimiter (;) before `box.commit()` so it's never called as `s:get{1}` now fails instead of returning `nil`. Fixed it with the following patch.

<details><summary>diff</summary>

```diff
diff --git a/test/box/transaction.result b/test/box/transaction.result
index 2a34e2e5a308..d4cc01712fb0 100644
--- a/test/box/transaction.result
+++ b/test/box/transaction.result
@@ -211,9 +211,14 @@ box.commit();
 box.begin() s:insert{1, 'Must be rolled back'};
 ---
 ...
--- nothing - the transaction was rolled back
--- nothing to commit because of yield
-while s:get{1} ~= nil do fiber.sleep(0) end
+fiber.yield();
+---
+...
+-- error - the transaction was rolled back by yield
+s:get{1};
+---
+- error: Transaction has been aborted by a fiber yield
+...
 box.commit();
 ---
 - error: Transaction has been aborted by a fiber yield
diff --git a/test/box/transaction.test.lua b/test/box/transaction.test.lua
index a1e49f5ac2b3..6bd039f65ac7 100644
--- a/test/box/transaction.test.lua
+++ b/test/box/transaction.test.lua
@@ -99,9 +99,9 @@ box.begin();
 -- back a transction with no statements.
 box.commit();
 box.begin() s:insert{1, 'Must be rolled back'};
--- nothing - the transaction was rolled back
-while s:get{1} ~= nil do fiber.sleep(0) end
--- nothing to commit because of yield
+fiber.yield();
+-- error - the transaction was rolled back by yield
+s:get{1};
 box.commit();
 -- Test background fiber
 --
```

</details>